### PR TITLE
Artwork packages (corrected)

### PR DESCRIPTION
**Pull request by moondeck**
Like #1096 but corrected (thanks to @MartijnBraam for guiding me through git)

### CODE REVIEW

File: aports/main/postmarketos-artwork-wallpapers/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Contributor: Olgierd "moondeck" Nowakowski <olgierd@novakovsky.eu>
+# Maintainer: Olgierd "moondeck" Nowakowski <olgierd@novakovsky.eu>
+
+_hash="2cad2af027fbec6aef0d1f16226a41ccf427f44e"
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork-wallpapers/APKBUILD]: I recommend to put the `_hash` variable futher below (e.g. above `source`) to be consistent with the other `APKBUILD`s.

File: aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pkgrel=0
+pkgdesc="Wallpapers for postmarketOS"
+url="http://postmarketos.org/"
+arch="noarch"
+license="CC-BY-SA"
+source="postmarketos-artwork-$_hash.tar.gz::https://github.com/postmarketOS/artwork/archive/${_hash}.tar.gz"
+options="!check"
+
+package() {
+  mkdir -p "$pkgdir"/usr/share/wallpapers/
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD]: We use tabs in `APKBUILD`s to be consistent with Alpine's `APKBUILD`s. (The Python code uses spaces to follow PEP8.)

File: aports/main/postmarketos-artwork-wallpapers/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+pkgrel=0
+pkgdesc="Wallpapers for postmarketOS"
+url="http://postmarketos.org/"
+arch="noarch"
+license="CC-BY-SA"
+source="postmarketos-artwork-$_hash.tar.gz::https://github.com/postmarketOS/artwork/archive/${_hash}.tar.gz"
+options="!check"
+
+package() {
+  mkdir -p "$pkgdir"/usr/share/wallpapers/
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork-wallpapers/APKBUILD]: spaces instead of tabs

File: aports/main/postmarketos-artwork/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+pkgver=1
+pkgrel=0
+pkgdesc="meta-package for the postmarketOS artwork"
+url="http://postmarketos.org/"
+arch="noarch"
+license="CC-BY-SA"
+depends="postmarketos-artwork-wallpapers postmarketos-artwork-wallpapers-extra"
+
+package() {
+  mkdir -p "$pkgdir"
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork/APKBUILD]: spaces instead of tabs

File: aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD [hunk 5]
```diff
@@ -0,0 +1,22 @@
+# Contributor: Olgierd "moondeck" Nowakowski <olgierd@novakovsky.eu>
+# Maintainer: Olgierd "moondeck" Nowakowski <olgierd@novakovsky.eu>
+
+_hash="2cad2af027fbec6aef0d1f16226a41ccf427f44e"
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD]: I recommend to put the `_hash` variable futher below (e.g. above `source`) to be consistent with the other `APKBUILD`s.

File: aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+pkgname="postmarketos-artwork-wallpapers-extra"
+pkgver=1
+pkgrel=0
+pkgdesc="Wallpapers for postmarketOS"
+url="http://postmarketos.org/"
```
Comment (ollieparanoid) [inline aports/main/postmarketos-artwork-wallpapers-extra/APKBUILD]: Personally I prefer if we link to `https://` pages if they provide it.

**Comment by moondeck** [comment]
Terribly sorry for my bad "packaging", will correct it as soon as i have time, most likely tomorrow.

**Comment by ollieparanoid** [comment]
No need to be sorry, it was your first PR here! 😉

**Comment by moondeck** [comment]
Hey, actually second! https://github.com/postmarketOS/pmbootstrap/pull/1082 But yeah, i will correct it ASAP

File: aports/main/postmarketos-artwork-wallpapers/APKBUILD [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+package() {
+	mkdir -p "$pkgdir"/usr/share/wallpapers/
+	cp -r "$srcdir"/artwork-$_hash/wallpapers/logo/* \
+		"$pkgdir"/usr/share/wallpapers/
+}
+
+extra() {
+  mkdir -p "$subpkgdir"/usr/share/wallpapers/
+  cp "$srcdir"/artwork-$_hash/wallpapers/extra/* \
+    "$subpkgdir"/usr/share/wallpapers/
```
Comment (drebrez) [inline aports/main/postmarketos-artwork-wallpapers/APKBUILD]: @moondeck as explained in chat, in APKBUILD we should use tabs, not spaces.

Comment (moondeck) [reply aports/main/postmarketos-artwork-wallpapers/APKBUILD]: Weird, i've used "Tabify" on it (in Atom). I will submit a PR fixing that

**Pull request merged**